docs/_data/modals.json

```
@@ -0,0 +1,44 @@
+{
+  "classes": [
+    {
+      "class": ".s-modal",
+      "applies": "N/A",
+      "description": "Base parent container for modals"
+    },
+    {
+      "class": ".has-danger",
```

```suggestion
      "class": ".s-modal__danger",
```

We recently renamed this one